querybook/server/lib/query_executor/executors/trino.py

```diff
@@ -36,22 +36,23 @@ def _parse_exception(self, e):
         error_type = QueryExecutionErrorType.INTERNAL.value
         error_str = str(e)
         error_extracted = None
+
         try:
+            if isinstance(e, TrinoQueryError):
+                error_type = QueryExecutionErrorType.ENGINE.value
```

Comment:
there is some tricky behaviour of some exceptions nested from TrinoQueryError - they have no error_location (not like it is None, but like KeyError) 😞

Reply:
oh i meant that error_type is not used in the if statement, get_parsed_syntax_error returns an error type called `QueryExecutionErrorType.SYNTAX.value`. 
Similarly, error_str = e.message is not used since we are passing in e.message directly on line 46

also if there is no e.error_location for some of the TrinoQueryError, maybe we should add an if to check that?